drivers/scheduler/k8s/k8s.go

```diff
@@ -6371,15 +6376,47 @@ func (k *K8s) CSISnapshotTest(ctx *scheduler.Context, request scheduler.CSISnaps
 	dirtyData := "thisIsJustSomeRandomText"
 	snapName := request.SnapName
 	for i := 0; i < 3; i++ {
-		data := fmt.Sprint(dirtyData, strconv.Itoa(int(time.Now().Unix())))
-		err = k.writeDataToPod(data, pod.GetName(), pod.GetNamespace(), mountPath)
-		if err != nil {
-			return fmt.Errorf("failed to write data to restored PVC: %s", err)
-
-		}
-		err = k.snapshotAndVerify(size, data, fmt.Sprint(snapName, i), pod.GetNamespace(), storageClassName, request.SnapshotclassName, fmt.Sprint(request.RestoredPVCName, i), request.OriginalPVCName)
-		if err != nil {
-			return fmt.Errorf("failed to validate restored PVC content: %s ", err)
+		if *pvcObj.Spec.VolumeMode == corev1.PersistentVolumeBlock {
+			data := "this is pure volume rawblock test data"
```

Comment:
Please add separate method to writing data into raw block. Please move code there